quickstart/docker/minimal/compose.yml

```diff
@@ -0,0 +1,50 @@
+services:
+  initialize:
+    image: busybox
+    command: chown -Rc ${ZIGGY_UID:-1000} /persistent
+    user: root
+    environment:
+      HOME: /persistent
+      # PFXLOG_NO_JSON: "true"
+    volumes:
+    - ./persistent:/persistent
```

Comment:
this is compose, can the example make a volume and reference that instead of a mount like this?

Reply:
The primary use case I have in mind for this minimal quickstart example is an immediately up-and-running Ziti network to support Ziti edge development, i.e., things that run on Ziti, like edge SDKs and zrok.

We could provide a commented example of a named volume without changing the use case. One of my goals for the primary use case is a bind-mounted persistence directory to enable the developer to adjust the Ziti controller and router settings directly without the fuss of piping commands through Docker or starting over from scratch with new env var settings.

Reply:
I use this daily.

Reply:
I'd like people who download it to get a volume mount for their stuff. if they want to change it to a directory mount like this, i'd like to let them decide that.

Reply:
I removed from this branch the additional shortcut route that I'd proposed like `get.openziti.io/minimal/compose.yml`. The existing directory shortcut `get.openziti.io/dock` to `/quickstart/docker` in this repo already provides a getter URL like `get.openziti.io/dock/minimal/compose.yml`. 

As for named vs. bind volume mount, using a named volume works but requires an extra step that every adopter must perform before they can enjoy an instant Ziti network. I've documented the extra step in a comment inside the `compose.yml` like so.

```yaml
    volumes:
    # store the quickstart state on the Docker host in the same directory as this compose.yml file
    - ./persistent:/persistent
    # store the quickstart state in a named volume; you must comment the bind mount above, uncomment the named mount
    # below, and set permissions on the named volume if you choose this alternative, e.g.:
    #       docker compose run --entrypoint='' --user=0 quickstart chmod -Rc 1777 /persistent
    # - persistent:/persistent
```

The bind mount is the best default configuration for persistence because it doesn't require the extra step to set permissions on the named volume (set sticky bit as root so any UID can mkdir and thenceforth manage their own files).